client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -228,9 +236,11 @@ const Checkout = React.createClass( {
 
 			return purchasePaths.managePurchase( renewalItem.extra.purchaseDomain, renewalItem.extra.purchaseId );
 		} else if ( cartItems.hasFreeTrial( this.props.cart ) ) {
-			this.props.clearSitePlans( this.props.sites.getSelectedSite().ID );
+			this.props.clearSitePlans( selectedSiteId );
 
-			return `/plans/${ this.props.sites.getSelectedSite().slug }/thank-you`;
+			return selectedSiteSlug
+				? `/plans/${ selectedSiteSlug }/thank-you`
+				: '/plans/';
```

Comment:
I'm not too happy with this, it avoids returning `/plans/undefined/thank-you`... so it's a slight improvement, but the UX of just kicking back to the plans page isn't ideal.
If we were to land these changes, would it be a good idea to open an issue to replace these defaulting paths?